SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -145,37 +133,20 @@ void USpatialDispatcher::ProcessExternalSchemaOp(Worker_Op* Op)
 	Worker_ComponentId ComponentId = GetComponentId(Op);
 	check(ComponentId != SpatialConstants::INVALID_COMPONENT_ID);
 
-	UOpCallbackTemplate** UserCallbackWrapper = UserOpCallbacks.Find(ComponentId);
-	if (UserCallbackWrapper == nullptr)
-	{
-		return;
-	}
-	UOpCallbackTemplate* UserCallback = *UserCallbackWrapper;
-
 	switch (Op->op_type)
 	{
+	case WORKER_OP_TYPE_AUTHORITY_CHANGE:
+		StaticComponentView->OnAuthorityChange(Op->authority_change);
```

Comment:
```suggestion
		StaticComponentView->OnAuthorityChange(Op->authority_change);
		// Intentional fall-through
```

Reply:
@alastairdglennie this comment can be resolved now right?